.travis.yml

```diff
@@ -25,3 +25,4 @@ matrix:
 
 script:
   - cargo test --all -- --nocapture
```

Comment:
Is there a reason not to have failpoints part of the normal `test`?

Reply:
Because failpoints injest failure in code path, which will affect all concurrently running tests, Hence they need to be synchronized, which make tests slow.

Reply:
:+1: Great reasoning, can you please add that in a comment?